core/src/main/scala/org/http4s/Static.scala

```diff
@@ -163,12 +167,12 @@ object Static {
     def fromFile(f: File,
                  blockingExecutionContext: ExecutionContext,
                  start: Long = 0L,
-                 end: Option[Long] = None,
+                 length: Option[Long] = None,
                  buffsize: Int = 8192,
                  req: Option[Request[F]] = None,
                  etagCalculator: File => F[String] = calcETag _): OptionT[F, Response[F]] =
       OptionT(for {
-        fileEndSize <- end.fold(F.delay(f.length()))(_.pure[F])
+        fileEndSize <- length.fold(F.delay(f.length()))(_.pure[F])
```

Comment:
Since we changed this to `length`, I'd expect to see some corresponding logic that adds it to start... or else we were confusing `end` with `start` before.  But this one is not just a simple rename.